libraries/matrix/impl/src/test/kotlin/io/element/android/libraries/matrix/impl/fixtures/fakes/FakeFfiTimelineEvent.kt

```diff
@@ -14,7 +14,7 @@ import org.matrix.rustcomponents.sdk.NoPointer
 import org.matrix.rustcomponents.sdk.TimelineEvent
 import org.matrix.rustcomponents.sdk.TimelineEventType
 
-class FakeFfiTimelineEvent(
+open class FakeFfiTimelineEvent(
```

Comment:
An other option would have to replace 
`val timelineEventType: TimelineEventType = aRustTimelineEventTypeMessageLike()`
by something like 
`val timelineEventTypeResult: () -> TimelineEventType =  { aRustTimelineEventTypeMessageLike() }`

But OK for this shortcut.